### DISCUSSION TITLE
Quiz_03

### DISCUSSION THREAD

**Discussion by dan21014** [cooperrc/advanced-dynamics#32] [Q&A]
So from quiz_02, we know that the motion of the collar is simply, r=x (i). Does this mean that the work of the spring is just equal to the motion of the collar multiplied by the force of the spring W_s = -kx * x (i) ?
Furthermore, assuming a frictionless system, wouldn't the work due to gravity equal the work of the spring, or technically be inversely proportional making the work of the spring negative?

**Comment by RebeccaVillanueva** [comment]
The spring force is in the opposite direction of gravity, since the spring is trying to pull mass 1 back up towards the ceiling when it's stretched past it's natural, at-rest position
